dashboard.el

```diff
@@ -440,6 +445,7 @@ Optional argument ARGS adviced function arguments."
   "Execute BODY in dashboard buffer."
   (declare (indent 0))
   `(with-current-buffer (get-buffer-create dashboard-buffer-name)
+     (run-hooks 'dashboard-before-initialize-hook)
```

Comment:
I don't think this should be inserted here.

Reply:
should i move this into dashboard-insert-startupify-lists?

this hook must be run before dashboard inserts any widgets.

Reply:
Yes.